src/editor/serialize.ts

```diff
@@ -55,22 +55,26 @@ export function htmlSerializeIfNeeded(model: EditorModel, {forceHTML = false} =
         // conditions for display math detection $$...$$:
         // - pattern starts at beginning of line or is not prefixed with backslash or dollar
         // - left delimiter ($$) is not escaped by backslash
-        const displayPatternDollar = "(^|[^\\\\$])\\$\\$(([^$]|\\\\\\$)+?)\\$\\$";
+        const displayPatternAlternative = (SdkConfig.get()['latex_maths_delims'] ||
+            {})['display_pattern_alternative'] ||
```

Comment:
What does "alternative" mean here...? Sorry, I think I am missing some context... 😅

Reply:
What I meant was `display_pattern` is the "default" pattern (example: `\[...\]`) for detecting math and `display_pattern_alternative` is an alternative (or additional) pattern (example: `$$...$$`) to do the same, but using a different syntax. I hope that makes sense 😂